app/views/merch/edit.ejs

```diff
@@ -0,0 +1,53 @@
+<!--
+Copyright © 2017, ACM@UIUC
+
+This file is part of the Groot Project.  
+
+The Groot Project is open source software, released under the University of
+Illinois/NCSA Open Source License.  You should have received a copy of
+this license in a file with the distribution.
+-->
+
+<%- include('../_partials/header') -%>
+
+<div id="edit-item-container">
+  <div class="row">
+    <% if ( errors.length > 0 ) { %>
```

Comment:
Can you use the flash partial here? https://github.com/acm-uiuc/groot-desktop-frontend/blob/master/app/views/_partials/flash.ejs